src/app/store.js

```diff
@@ -3,13 +3,13 @@ import { combineReducers, createStore } from "redux";
 import { firestoreReducer } from "redux-firestore";
 
 // Add firebase to reducers
-const rootReducer = combineReducers({
+export const rootReducer = combineReducers({
```

Comment:
I wan't to say that there's a way to mock this in the tests so we don't have to export this...
Have you looked at the other test files to see how the store is created in the test environment?


@mat10tng or @qdozaq might have better insight.

Reply:
I removed `export` from both `rootReducer` and `initialState`. I also updated the test for MissionControl.